Client/mods/deathmatch/logic/lua/CLuaArguments.cpp

```diff
@@ -293,82 +286,79 @@ bool CLuaArguments::CallGlobal(CLuaMain* pLuaMain, const char* szFunction, CLuaA
 
         return false;            // the function call failed
     }
-    else
-    {
-        int iReturns = lua_gettop(luaVM) - luaStackPointer;
+    int iReturns = lua_gettop(luaVM) - luaStackPointer;
 
-        if (returnValues != NULL)
+    if (returnValues)
+    {
+        for (int i = -iReturns; i <= -1; i++)
```

Comment:
int32_t maybe?

Reply:
Doesn't really matter for normal `int` but sure, to keep "compatibility"